Documentation/devel/stage1-implementors-guide.md

```diff
@@ -63,18 +62,18 @@ rkt supports two ways of doing that, stage1 implementors must pick one:
 * `--interactive` to run a pod interactively, that is, pass standard input to the application (only for pods with one application)
 * `--local-config=$PATH` to override the local configuration directory
 
-### `rkt enter` => "coreos.com/rkt/stage1/enter"
+### `rkt enter` => `coreos.com/rkt/stage1/enter`
 
 0. rkt verifies the pod and image to enter are valid and running
-1. chdirs to "/var/lib/rkt/pods/$uuid"
-2. resolves the "coreos.com/rkt/stage1/enter" entrypoint via Annotations found within "/var/lib/rkt/pods/$uuid/stage1/manifest"
-3. executes the resolved entrypoint relative to "/var/lib/rkt/pods/$uuid/stage1/rootfs"
+1. chdirs to `/var/lib/rkt/pods/$uuid`
```

Comment:
Suggest: Renumber this list to be index-1. (I believe markdown will render it index-1 in HTML, anyway)